extract/src/main/resources/static/lang/de/rulesHelp.html

```diff
@@ -0,0 +1,481 @@
+
+            <td class="popover-table-data">
+                <p>Bezeichnung des bestellten Produkts</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Text</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>client</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Kunde, der das Anfrageelement bestellt hat</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Text</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>clientguid</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Identifikator des Kunden, der das Anfrageelement bestellt hat</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Text</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>tiers</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Dritter, für den das Anfrageelement bestellt wurde</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Text</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>tiersguid</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Identifikator des Dritten, für den das Anfrageelement bestellt wurde</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Text</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>surface</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Fläche des Umfangs des Anfrageelements in m²</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Numerisch</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>organism</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Organismus des Kunden, der das Anfrageelement bestellt hat</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
```

Comment:
I am not sure if it is correct that elements such as "Attributiv" and "Numerisch" here have been translated to german. Whereas elements such as "organismusguid" have not.
Are these actually keys that are used somewhere in the code of the application, and so they should not be translated?

Reply:
I think, here "Attributiv" or "Numerisch" do not refer to an actual types, just human words for "this should be a text label", "here should be a number"

But "organism" and "organismguid" are used in an actual code. Most important example is "[Request.java](https://github.com/camptocamp/extract/blob/master/extract/src/main/java/ch/asit_asso/extract/domain/Request.java)"